models/Document/PageSnippet.php

```diff
@@ -60,9 +60,10 @@ abstract class PageSnippet extends Model\Document
      *
      * @var array
      *
-     * @deprecated since v6.7 and will be removed in Pimcore 10. Use getter/setter methods or $this->editables
+     * @deprecated since v6.7 and will be removed in 10. Use getter/setter methods getEditables/setEditables instead.
+     *                From 6.9 on this property will be private instead of protected.
```

Comment:
Does this make sense? Actually it is already `private` here.

It should probably rather be added in the `6.8` branch.

(And it's missing for `$inheritedElements`)

Reply:
We thought `6.9` would be better, as it potentially could have an impact on existing project related code.

Reply:
What I meant was, that only the message `From 6.9 on this property will be private instead of protected.` goes into the branch `6.8` while the actual change goes into `6.9.` (like it is actually targeted).

Reply:
@jdreesen valid and good point 👍

Reply:
@jdreesen see https://github.com/pimcore/pimcore/pull/7706/commits/bffe7251107cba22d8a2730587dfbc7e3d860ddd